Update minimum role for alerts to member (#2876)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx

description: "Learn more about creating and managing Issue Alerts."
---

> Create and manage Issue Alerts by navigating to Alerts and clicking the "Alert Rules" tab, where you'll see a list of all active rules and can add new rules or modify existing ones. Then, select your project and choose Issue Alert.

Reviewer: ```suggestion
Create and manage Issue Alerts by navigating to Alerts and clicking the "Alert Rules" tab, where you can both view a list of all active rules and add new rules or modify existing ones. Then, select your project and choose Issue Alert.
```

> The minimum role required to create alerts is member. Sentry users with manager or owner permissions can change this to admin in **Settings > General Settings > Grant Members Alerts Write**.

Reviewer: ```suggestion
The minimum role required to create alerts is member. Sentry users with manager or owner permissions can change the minimum role requirement in **Settings > General Settings > Grant Members Alerts Write**.
```
